<launch>
    # convert
    <node name="nclt2rosbag" pkg="nclt2ros" type="nclt2rosbag" output="screen">

        <param name="date"                  value="2012-04-29" />
        <param name="raw_data_path"         value="/home/entc/kalani/data/nclt" />
        <param name="rosbag_output_path"    value="/home/entc/kalani/data/nclt" />
        <param name="bag_name"              value="nclt_mod" />
        <param name="cam_folder"            value="5" />

        # specify data to convert
        <param name="lb3"                   value="False" />
        <param name="sen"                   value="True" />
        <param name="vel"                   value="True" />
        <param name="hokuyo"                value="False" />
        <param name="gt"                    value="True" />

        # topic names
        <param name="gps_fix"               value="/raw_data/gnss_fix" />
        <param name="gps_track"             value="/raw_data/track" />
        <param name="gps__speed"            value="/raw_data/speed" />
        <param name="gps_rtk_fix"           value="/raw_data/rtk_gnss_fix" />
        <param name="gps_rtk_track"         value="/raw_data/rtk_gnss_track" />
        <param name="gps_rtk_speed"         value="/raw_data/rtk_gnss_speed" />
        <param name="ms25_imu_data"         value="/raw_data/imu" />
        <param name="ms25_imu_mag"          value="/raw_data/mag" />
        <param name="wheel_odometry_topic"  value="/raw_data/odom" />
        <param name="hokuyo_utm_topic"      value="/raw_data/hokuyo_30m" />
        <param name="hokuyo_urg_topic"      value="/raw_data/hokuyo_4m" />
        <param name="velodyne_topic"        value="/raw_data/velodyne_points" />
        <param name="ladybug_topic"         value="/raw_data/images" />
        <param name="ground_truth_topic"    value="/raw_data/ground_truth" />

        # frame ids
        <param name="gps_sensor"            value="gps_link" />
        <param name="gps_rtk_sensor"        value="gps_rtk_link" />
        <param name="imu_sensor"            value="imu_link" />
        <param name="wheel_odometry_sensor" value="odom" />
        <param name="hokuyo_utm_lidar"      value="laser_utm" />
        <param name="hokuyo_urg_lidar"      value="laser_urg" />
        <param name="velodyne_lidar"        value="velodyne" />
        <param name="ladybug_sensor"        value="camera" />
        <param name="ground_truth"          value="ground_truth_link" />
        <param name="body"                  value="base_link" />

    </node>
</launch>
